packages/twenty-front/src/modules/object-record/record-field/hooks/usePersistField.ts

```diff
@@ -137,15 +146,20 @@ export const usePersistField = () => {
           );
 
           if (fieldIsRelation) {
-            updateRecord?.({
-              variables: {
-                where: { id: entityId },
-                updateOneRecordInput: {
-                  [fieldName]: valueToPersist,
-                  [`${fieldName}Id`]: valueToPersist?.id ?? null,
+            if (fieldIsRelationFromManyObjects) {
+              throw new Error('Cannot update this relation.');
```

Comment:
we are not using usePersistField for FromManyObjects as it requires a specific logic that did not feel right to add here

Reply:
It feels weird to throw here. You have a boolean `isValuePersistable` above that is true if `fieldIsRelation`. Can you simply change for `fieldIsRelation && ! fieldIsRelationFromManyObjects`?